test/e2e/v3/plugin_cluster_test.go

```diff
@@ -17,7 +17,6 @@ limitations under the License.
 package v3
```

Comment:
We use the emojis to create the changelog.
This change is only in the tests and does not affect the end-users
So that is not a bug we need change it for 🌱 , see the contribution guide.